commit.py

```
@@ -9,133 +9,191 @@
+        print(f"Error processing .docx file: {e}")
+        sys.exit(1)
     print(f"Error processing .docx file: {e}")
     sys.exit(1)
+    return hashed_file
+
+def convert_docx_to_html():
+    try: 
+        with open(path, "rb") as f:
+            result = mammoth.convert_to_html(f)
+            html = result.value
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)